source/reference/limitations.txt

```diff
@@ -64,6 +64,36 @@ General Limitations
 - You cannot sync a collection that has a unique index and a non-unique
   index defined on the same field(s).
 
+.. _c2c-limitations_multiple_sync:
+
+Multiple Source Sync Limitations
+--------------------------------
+
+.. include:: /includes/many-with-one-cluster.rst
+
+A namespace is a ``database_name.collection_name`` combination. You can
+only sync namespaces that don't conflict.
+
+For example, consider this scenario:
+
+- Two source clusters S1 and S2.
+- A destination cluster D.
+- Databases ``X`` and ``Y`` on both S1 and S2.
+- Collections ``A``, ``B``, ``C``, and ``D`` on both S1 and S2.
```

Comment:
**Suggestion**

You might consider some longer collection names of a standard database schema such as `orders`, `orderlines`, `orderStatus`, `Product`. It is slightly hard to parse the concept here because the collection and database names are so short and unrelatable.

Reply:
Will do.